biblib/views/user_view.py

```diff
@@ -200,17 +208,27 @@ def get(self):
             get_params = request.args
             start = get_params.get('start', default=0, type=int)
             rows = get_params.get('rows', type=int)
+            
+            sort_col = get_params.get('sort', default='date_created', type=str)
+            if sort_col not in ['date_created', 'date_last_modified']: 
+                raise ValueError
+            
+            sort_order = get_params.get('order', default='desc', type=str)
```

Comment:
Let's discuss this default sort order - I had thought/assumed we'd do asc by default, so the earlier results would be unchanging, but I can see the arguments for seeing the most recent libraries first instead. Let's discuss this point in person this week.